server/master/src/main/java/org/apache/accumulo/master/replication/RemoveCompleteReplicationRecords.java

```diff
@@ -57,9 +58,11 @@
   private static final Logger log = LoggerFactory.getLogger(RemoveCompleteReplicationRecords.class);
 
   private Connector conn;
+  private Master master;
 
-  public RemoveCompleteReplicationRecords(Connector conn) {
+  public RemoveCompleteReplicationRecords(Connector conn, Master master) {
```

Comment:
Is this connector for the remote peer? If not, the master actually contains a client context and the connector is redundant if the master is being passed in.